### DISCUSSION TITLE
Configuration and selection capability for multiple RustDesk server endpoints

### DISCUSSION THREAD

**Discussion by mcrainydrop** [rustdesk/rustdesk#4694] [General]
Hello together,

with RustDesk it's possible to use a self-hosted server for establishing the remote connection.

With this capability, I have my own RustDesk server and my friends also have their own ones.
Extending this scenario, in a IT-support or helpdesk setup, multiple customers can have their own RustDesk servers.

It would be great, if the RustDesk client interface provide the capabilities to configure not just one server (with its endpoint URL and key), but mulitple servers which can be selected e.g. using a selectbox for easier switching between multiple server backends ?!

Have a nice day

**Comment by yanis761** [comment]
In my scenario
- I have my own server
- Two clients have their server
- Other uses pulic server

I have to continuously select different network settings
It is also impossible to connect to different clients at the same time

**Comment by thebaldgeek** [comment]
This request is pretty much the same here:
https://github.com/rustdesk/rustdesk/discussions/6922

We are encouraged to build and host our own servers, but then switching between them is a major pain (but for a simple multi selector switch in the UI).

The copy/paste option was added to the server UI in the past few updates, but is useless as when you come back from connecting to another client, the clipboard has been wiped, so you have to set it all up again each time.

**Reply by MagicFab** [reply]
The link for the discussion doesn't work now.

**Comment by HomeWiseAdmin** [comment]
Please implmenet choosing the network config in the UI or in a config file.   Thanks, Lets upvote this as a feature request

**Comment by MagicFab** [comment]
It seems this discussion should happen in #1258

**Comment by boutzamat** [comment]
Still no solution? I just purchased Pro, to realize i can't even have both my own private config (for my personal devices) and switch to a work profile that runs through my Pro version. Very frustrating
